src/services/response-service.ts

```diff
@@ -31,15 +31,27 @@ export default class ResponseService extends PercyClientService {
     const parsedRootResourceUrl = new URL(rootResourceUrl)
     const rootUrl = `${parsedRootResourceUrl.protocol}//${parsedRootResourceUrl.host}`
 
-    if (
+    if (request.url() === rootResourceUrl) {
       // Always skip the root resource
-      request.url() === rootResourceUrl
+      logger.debug(`Skipping [is_root_resource]: ${response.url()}`)
+      return
+    }
+
+    if (!this.ALLOWED_RESPONSE_STATUSES.includes(response.status())) {
       // Only allow 2XX responses:
-      || !this.ALLOWED_RESPONSE_STATUSES.includes(response.status())
-      || !request.url().startsWith(rootUrl) // Disallow remote resource requests.
-      || !response.url().startsWith(rootUrl) // Disallow remote redirects.
-      ) {
-      logger.debug(`Skipping: ${response.url()}`)
+      logger.debug(`Skipping [disallowed_response_status]: ${response.url()}`)
```

Comment:
In this case might be nice to log the response status we got.

Reply:
I was going to say the same thing :D

Reply:
included with bf131e59a7898e6f3afbd4cae85196ed1e52b62e